packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config-table.vue

```diff
@@ -181,6 +181,10 @@ const matrixModalContext = ref({
 	matrixId: ''
 });
 
+const modelType = computed(
+	() => props.modelConfiguration?.configuration?.header?.schema_name ?? AMRSchemaNames.PETRINET
+);
```

Comment:
I think we need to create a util `isPetrinet` because we need to test the values `toLowerCase()`

Reply:
ah, Ill add a function called like 'getModelType' and we can pass in the model and it returns the schema_name with toLowercase on it

Reply:
added everywhere we get the model type, defaults to petrinet